index.bs

```diff
@@ -3638,6 +3626,9 @@ Methods</h4>
 		are also removed if the hold time occurs after
 		<code>cancelTime</code>.
 
+		If <code>cancelTime</code> is 0, this method removes all events except the
+		initial event inserted on construction of the {{AudioParam}}.
+
```

Comment:
I don't like this exception. I also don't immediately see any way around this.

Reply:
1. When the AudioParam is constructed, setValue(val, 0) is called.
2. We call cancelAndHold(0).
3. The "current AudioParam event" is the one that's used to compute the value at time 0, i.e. the initial setValue.
4. CancelAndHold() removes all events that are in the event list **after** the current AudioParam event. That means it doesn't remove the event itself.

It looks like the special case is not necessary.